test/nn/test_convolution.py

```diff
@@ -1927,6 +1927,7 @@ def test_conv_noncontig_weights_and_bias(self, device):
 
     @onlyCUDA
     @largeTensorTest('12GB')
+    @skipIfRocmVersionLessThan((6, 0))
```

Comment:
Just to be clear, this feels like it skips it if ROCm version is older than X rather then if runner type is older, isn't it? So title is a bit misleading

Reply:
I agree it's a bit confusing, but the failure is a MI210-specific failure which will be fixed in ROCm6.0, hence the `skipIfRocmVersionLessThan` decorator